runtime/ISO_Fortran_binding.cc

```diff
@@ -20,6 +20,13 @@
 namespace Fortran::ISO {
 extern "C" {
 
+static inline constexpr bool IsCharacterType(CFI_type_t ty) {
+  return ty == CFI_type_char;
+}
+static inline constexpr bool IsAssumedSize(const CFI_cdesc_t *dv) {
+  return dv->rank > 0 && dv->dim[dv->rank - 1].extent == -1;
+}
+
 void *CFI_address(
     const CFI_cdesc_t *descriptor, const CFI_index_t subscripts[]) {
   auto p = reinterpret_cast<char *>(descriptor->base_addr);
```

Comment:
prefer static_cast<>

Reply:
Thanks. I took the opportunity to also change the size_t types in this function because I do not see good reasons to use size_t for the rank.